forest/sycamore/sycamore_main.py

```diff
@@ -1,48 +1,109 @@
 import os
-from .functions import aggregate_surveys_config, aggregate_surveys_no_config
-from .survey_config import survey_submits, survey_submits_no_config
-from .changed_answers import agg_changed_answers_summary
+from typing import Optional, List
 
+from forest.sycamore.functions import (aggregate_surveys_config,
+                                       aggregate_surveys_no_config)
+from forest.sycamore.survey_config import (survey_submits,
+                                           survey_submits_no_config,
+                                           get_all_interventions_dict)
+from forest.sycamore.changed_answers import agg_changed_answers_summary
 
-def survey_stats_main(output_dir, study_dir, beiwe_ids, config_path=None, time_start=None, time_end=None,
-                      study_tz=None):
+
+def survey_stats_main(
+        study_folder: str,
+        output_folder: str,
+        tz_str: Optional[str] = "America/New_York",
+        participant_ids: Optional[List] = None,
+        time_start: Optional[str] = None,
+        time_end: Optional[str] = None,
+        config_path: Optional[str] = None,
+        interventions_filepath: Optional[str] = None) -> bool:
     """
+    Compute statistics on surveys.
+
     Args:
-    output_dir(str):
+    output_folder(str):
         File path to output summaries and details
-    study_dir(str):
+    study_folder(str):
         File path to study data
     config_path(str):
         File path to study configuration file
     time_start(str):
         The first date of the survey data
     time_end(str):
         The last date of the survey data
-    beiwe_ids(list):
+    participant_ids(list):
         List of users in study for which we are generating a survey schedule
-    study_tz(str):
+    tz_str(str):
         Timezone of study. This defaults to 'America/New_York'
-    """
+    interventions_filepath(str):
+        filepath where interventions json file is.
 
+    """
+    if not os.path.isdir(output_folder):
+        os.mkdir(output_folder)
```

Comment:
No need for a conditional, simply: `os.makedirs(output_folder, exist_ok=True)`
https://docs.python.org/3.8/library/os.html#os.makedirs

Reply:
Ah, thanks!